### PR TITLE
Add bots documentation

### PR DESCRIPTION
**Pull request by ldez**
### Description

- add bots documentation
- move `CONTRIBUTING.md` and `MAINTAINER.md` from `.github` to the root.
- suppress PR scripts (cpr, pfpr, rmpr, rpr)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ The title must be short and descriptive. (~60 characters)
 
 ### Description
 
-- Respect the issue template as more as possible. [template](ISSUE_TEMPLATE.md)
+- Respect the issue template as more as possible. [template](.github/ISSUE_TEMPLATE.md)
```
Comment (timoreimann) [inline CONTRIBUTING.md]: grammar: _as much as possible_.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -205,7 +205,7 @@ The title must be short and descriptive. (~60 characters)
 
 ### Description
 
-- Respect the pull request template as more as possible. [template](PULL_REQUEST_TEMPLATE.md)
+- Respect the pull request template as more as possible. [template](.github/PULL_REQUEST_TEMPLATE.md)
```
Comment (timoreimann) [inline CONTRIBUTING.md]: s/more/much/ here too.

File: MAINTAINER.md [hunk 3]
```diff
@@ -12,6 +12,46 @@
 - Julien Salleyron [@juliens](https://github.com/juliens)
 - Nicolas Mengin [@nmengin](https://github.com/nmengin)
 
+
+## PR review process:
+
+* If needed, a specific maintainer can be required to make a review (according to its knowledge areas)
```
Comment (timoreimann) [inline MAINTAINER.md]: I'd rephrase this to

_If needed, a specific maintainer familiar with a particular domain can be requested for the review._

I'd also move the remark after the next block (i.e., between lines 21 and 23) since it's more of an add-on information to the more central description of how many LGTMs are needed.

File: MAINTAINER.md [hunk 4]
```diff
@@ -12,6 +12,46 @@
...
+We use [PRM](https://github.com/ldez/prm) to manage locally pull requests.
+
+
+## Bots
+
+### [Myrmica Lobicornis](https://github.com/containous/lobicornis/)
+
+**Update and Merge Pull Request**
+
+As maintainer when you add the third LGTM, you must add `status/3-needs-merge`.
```
Comment (timoreimann) [inline MAINTAINER.md]: Wording improvement suggestion:

_The maintainer giving the final LGTM must add the `status/3-needs-merge` label to trigger the merge bot._

File: MAINTAINER.md [hunk 5]
```diff
@@ -12,6 +12,46 @@
...
+If you want to preserve commits you must add `bot/merge-method-rebase` before `status/3-needs-merge`.
```
Comment (timoreimann) [inline MAINTAINER.md]: I'd prepend this sentence by

_By default, a squash-rebase merge will be carried out._

Comment (timoreimann) [inline MAINTAINER.md]: Shall we add a note about the exceptions as to when vendor changes are involved? AFAICS, we do it as described followingly:

_If a PR involves changes to third-party dependencies, the commits pertaining to the `vendor` folder and the manifest/lock file(s) should be committed separated, respectively. In such cases, the rebase merge method should be selected and any squash-worthy commits be squashed manually by the author(s)._

File: MAINTAINER.md [hunk 6]
```diff
@@ -12,6 +12,46 @@
...
+
+The status `status/4-merge-in-progress` is only for the bot.
+
+If the bot is not able to perform the merge, he add `bot/need-human-merge`.
```
Comment (timoreimann) [inline MAINTAINER.md]: s/he add/it adds/.

File: MAINTAINER.md [hunk 7]
```diff
@@ -12,6 +12,46 @@
...
+In this case you must solve conflicts/CI/... and after you only need to remove `bot/need-human-merge`.
+
+
+### [Myrmica Bibikoffi](https://github.com/containous/bibikoffi/)
+
+- closes stale issues [cron]
```
Comment (timoreimann) [inline MAINTAINER.md]: Shall we describe the circumstances that define when an issue becomes stale? I personally don't know them.

Comment (ldez) [inline MAINTAINER.md]: For now, we can not describe the rules we have not defined 😄

File: MAINTAINER.md (hunk 3 above)
Comment (ldez) [inline MAINTAINER.md]: Do you know that some reviews concern things I don't have change?

File: MAINTAINER.md (hunk 5 above)
Comment (ldez) [inline MAINTAINER.md]: I put the first sentence in the contributing guide.
I don't agree with the second sentence.

File: MAINTAINER.md (hunk 3 above)
Comment (timoreimann) [inline MAINTAINER.md]: I'm aware of that. That shouldn't keep us from being good boy scouts and doing drive-by refactorings where we think it's helpful.

File: MAINTAINER.md (hunk 7 above)
Comment (timoreimann) [inline MAINTAINER.md]: Then I think we should add some note saying that this very bot is still WIP or so. Right now, it looks a bit like we forgot to document it.

Comment (ldez) [inline MAINTAINER.md]: I understand, but after reflexion I don't think we're going to explain the rules in detail here: the rules can change and I don't want to make a PR for every change we're going to make.
I add a quick explanation.

Comment (timoreimann) [inline MAINTAINER.md]: Works for me. :+1:
